Client/Ecosia/Analytics/Analytics+Configuration.swift

```diff
@@ -13,7 +13,7 @@ extension Analytics {
         .sessionContext(true)
         .applicationContext(true)
         .platformContext(true)
-        .platformContextProperties([]) // track minimal device properties
+        .platformContextProperties([.appleIdfv]) // track minimal device properties
```

Comment:
Is `// track minimal device properties` still valid? I guess we could change to `.appleIdfv` needed for MMP tracking

Reply:
Yeah why not 👍 
I thought that comment was mostly as a sort of reminder, which at that time meant "nothing" regarding platforms.